### ISSUE TITLE
IDE preferences window's vertical size is too small

### ISSUE THREAD

**Issue by jamshark70** [supercollider/supercollider#4034]
Environment
-----------

* Your SuperCollider version: 3.10 beta
* Your operating system and version: Ubuntu Studio 16.04

Issue
-----

Minor issue, probably very easy for someone to fix -- but it is a usability issue and it looks quite unprofessional.

The IDE's Preferences window is too small for the interpreter panel to be usable.

Note below the size of the "include paths" scrollbar. Guess what happens when you try to grab the handle and scroll... that's right. Nothing.

So the workflow, to remove an included path, includes an extra step: double-click the title bar to expand to the entire screen size.

Could be solved easily by increasing the vertical size of this window.

![preferences](https://user-images.githubusercontent.com/318301/44886486-f3fb4e00-acf8-11e8-9b9f-fd53362566f0.png)

**Comment by brianlheim** [comment]
@jamshark70 About how tall should the window be for it to be usable? Can you do a measurement? I'm not quite sure I understand what the problem is, so it's probably just easiest if you give a concrete number.

**Comment by jamshark70** [comment]
TL;DR According to the Linux screen ruler utility, about 500 pixels vertically should do it. The next round number in binary is 512, so, how about that?

The problem is that in the vertical scroll bars (for both the include paths and exclude paths), the handle seems to have a minimum height (seems to be about 25 pixels).

Also, there is only so much vertical space available. I'm not sure exactly how that space is divided up, but on my system, the "include" path list is slightly shorter than the "exclude" list.

Within the "include" scrollbar, excluding the up and down arrow widgets, there are only 25 or 26 pixels available for the handle. And the handle seems to require a minimum of 25 pixels. So, the handle occupies all of the available space -- and this means there is nowhere to move the handle. The handle is frozen in the middle. There is nothing you can do with the mouse to make it move.

An alternate solution would be to require a larger minimum size for the list-box widgets. But I don't have a concrete suggestion for how to do that. At first glance, editors/sc-ide/forms/settings_sclang.ui seems to treat them equally, but on my system, with the default size, "include" shows three items and "exclude" shows (almost) four. Looks a bit dodgy tbh -- almost like we're accidentally suggesting that the exclude paths are more important. It would look better if we controlled the widget sizes better and made these two exactly the same.
